cpp/test/Glacier2/sessionControl/Client.cpp

```diff
@@ -52,14 +59,9 @@ SessionControlClient::run(int argc, char** argv)
     {
         test(false);
     }
-    try
-    {
-        session->ice_ping();
```

Comment:
In this version of the test, it's the ping to an unknown session via Glacier2 that makes the router close the connection forcefully.